### ISSUE TITLE
Reduce commit list flashing

### ISSUE THREAD

**Issue by Timmmm** [gitx/gitx#131]
Whenever you change branch or the repo is refreshed the commit revision history list, and the tree view (if you have it open) are cleared for about 0.3 seconds before they are filled again. Basically they flash.

It's no big deal but it's pretty visually annoying.

**Comment by tiennou** [comment]
Yes, it's an artifact of how the reloading is done. I'm filing that as an enhancement though because it's likely to require *a lot* of changes to make it more efficient.

**Comment by ADTC** [comment]
There are several problems with auto-refreshing, like this one. Currently the only workaround is to disable auto-refresh (Watch for changes) and refresh manually.
